tpu/config.txt

```diff
@@ -0,0 +1,3 @@
+BASE_IMAGE_TAG=v108
```

Comment:
This is the latest CPU build I tested this against that I believe works. Will bump it eventually.

Reply:
Can you add a TODO explaining this and the end goal of setting this to `ci-pretest`.